ui/src/app/views/concept-set-list/component.tsx

```diff
@@ -48,11 +43,8 @@ export const ConceptSetsList = withCurrentWorkspace()(
     }
 
     componentDidMount() {
-      try {
-        this.getDomainInfo();
-      } finally {
-        this.loadConceptSets();
-      }
+      this.getDomainInfo();
```

Comment:
Do we need to log errors if domainInfo returns error?

Reply:
We are, as part of the `getDomainInfo` function.